pkgs/applications/audio/curseradio/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, python3Packages, mpv }:
+
+python3Packages.buildPythonApplication rec {
+  version = "0.2";
+  name    = "curseradio-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "chronitis";
+    repo   = "curseradio";
+    rev    = "1bd4bd0faeec675e0647bac9a100b526cba19f8d";
+    sha256 = "11bf0jnj8h2fxhpdp498189r4s6b47vy4wripv0z4nx7lxajl88i";
+  };
+
+  propagatedBuildInputs = with python3Packages;
+  [ requests lxml pyxdg];
+
+  postPatch = ''
+    substituteInPlace curseradio/curseradio.py --replace /usr/bin/mpv ${mpv}/bin/mpv
```

Comment:
It's much preferred to make use of a patch for this.

Usually I do
```
patches = [
  (substituteAll {
    src = ./somename.patch;
    inherit mpv;
  }
];
```

The content of the patch should be replacing `/usr` with an environment variable that `substituteAll` can replace.
`@mpv@`

##### [link to example](https://github.com/NixOS/nixpkgs/blob/3e950d584c8186239809e4d101e70e083ea9e9e0/pkgs/applications/video/kazam/fix-paths.patch)